src/Raven.Server/Json/BlittableJsonTextWriterExtensions.cs

```diff
@@ -1192,6 +1193,38 @@ public static void WriteObjects(this BlittableJsonTextWriter writer, JsonOperati
             return numberOfResults;
         }
 
+        public static async Task WriteCountersAsync(this AsyncBlittableJsonTextWriter writer, JsonOperationContext context, List<CounterDetail> counters)
+        {
+            writer.WriteStartArray();
+
+            var first = true;
+            foreach (var counter in counters)
+            {
+                if (first == false)
+                    writer.WriteComma();
+                first = false;
+
+                writer.WriteStartObject();
+
+                writer.WritePropertyName(nameof(CounterDetail.DocumentId));
+                writer.WriteString(counter.DocumentId);
+                writer.WriteComma();
+
+                writer.WritePropertyName(nameof(CounterDetail.CounterName));
+                writer.WriteString(counter.CounterName);
+                writer.WriteComma();
+
+                writer.WritePropertyName(nameof(CounterDetail.TotalValue));
+                writer.WriteDouble(counter.TotalValue);
+
+                writer.WriteEndObject();
+
+                await writer.MaybeOuterFlsuhAsync();
```

Comment:
typo :)

I do not think we need to flush that?

Reply:
BTW there is typo there: Flsuh -> Flush

Reply:
fixed